telemetry.js

```diff
@@ -0,0 +1,146 @@
+// Collects anonymous usage stats and uncaught errors
```

Comment:
This file looks like it's only required from the renderer -- can you move it under `/renderer`?